pypeit/spectrographs/keck_nirspec.py

```diff
@@ -6,24 +7,1124 @@
+        
+        Returns:
+            :class:`~pypeit.par.pypeitpar.PypeItPar`: Parameters required by
+            all of PypeIt methods.
+        """
+        par = super().default_pypeit_par()
+
+        # Wavelengths
+        # 1D wavelength solution
+        par['calibrations']['wavelengths']['rms_thresh_frac_fwhm'] = 3.0 #0.20  # Might be grating dependent..
+        par['calibrations']['wavelengths']['sigdetect']=5.0
+        par['calibrations']['wavelengths']['fwhm']= 3.0
+        par['calibrations']['wavelengths']['fwhm_fromlines']= False
+        par['calibrations']['wavelengths']['n_final']= 4
+        par['calibrations']['wavelengths']['lamps'] = ['NIRSPEC-ArNeKrXe']#, 'ThAr']
+        #par['calibrations']['wavelengths']['nonlinear_counts'] = self.detector[0]['nonlinear'] * self.detector[0]['saturation']
+        par['calibrations']['wavelengths']['method'] = 'echelle'
+#        par['calibrations']['wavelengths']['ech_fix_format'] = True
+        par['calibrations']['wavelengths']['cc_thresh'] = 0.5
+        par['calibrations']['wavelengths']['cc_local_thresh'] = 0.50
+        # Echelle parameters
+        par['calibrations']['wavelengths']['echelle'] = True
+        par['calibrations']['wavelengths']['ech_nspec_coeff'] = 4
+        par['calibrations']['wavelengths']['ech_norder_coeff'] = 4
+        par['calibrations']['wavelengths']['ech_sigrej'] = 3.0
+        par['calibrations']['wavelengths']['ech_separate_2d'] = False
+
+        
+        # Trace ID parameters
+        par['calibrations']['slitedges']['edge_thresh'] = 100.0
+        par['calibrations']['slitedges']['fit_order'] = 4
+        par['calibrations']['slitedges']['max_shift_adj'] = 0.5
+        par['calibrations']['slitedges']['trace_thresh'] = 10.
+        par['calibrations']['slitedges']['left_right_pca'] = True
+        par['calibrations']['slitedges']['length_range'] = 0.3
+        par['calibrations']['slitedges']['max_nudge'] = 10.
+        par['calibrations']['slitedges']['overlap'] = True
+        par['calibrations']['slitedges']['dlength_range'] = 0.25
+
+        # Flats
+        par['calibrations']['flatfield']['tweak_slits_thresh'] = 0.80
+        par['calibrations']['flatfield']['slit_illum_relative'] = False#True
+
+        # Extraction
+        par['reduce']['skysub']['bspline_spacing'] = 0.8
+        par['reduce']['extraction']['sn_gauss'] = 4.0
+        par['reduce']['findobj']['fof_link'] = 0.2
+
+        # Flexure
+        par['flexure']['spec_method'] = 'skip'
+
+        par['scienceframe']['process']['sigclip'] = 20.0
+        par['scienceframe']['process']['satpix'] ='nothing'
+
+        # Should be we be illumflattening?
+
+        # Flats
+        turn_off = dict(use_biasimage=False, use_overscan=False,
+                        use_darkimage=False, use_specillum=False, use_illumflat = False, use_pixelflat=False) #use_illumflat=True, 
+        par.reset_all_processimages_par(**turn_off)
+
+        '''
```

Comment:
Please change the `'''` style comments to `#` style comments

Reply:
Thank you, I've removed those here